pkg/collector/prometheus_collector.go

```diff
@@ -618,15 +618,12 @@ func (p *PrometheusCollector) updateNodeMetrics(wg *sync.WaitGroup, ch chan<- pr
 
 // updatePodMetrics send pod metrics to prometheus
 func (p *PrometheusCollector) updatePodMetrics(wg *sync.WaitGroup, ch chan<- prometheus.Metric) {
-	const commandLenLimit = 10
 	for _, container := range *p.ContainersMetrics {
 		wg.Add(1)
 		go func(container *collector_metric.ContainerMetrics) {
 			defer wg.Done()
-			containerCommand := container.Command
-			if len(containerCommand) > commandLenLimit {
-				containerCommand = container.Command[:commandLenLimit]
-			}
+			// removed command because issue #956
+			containerCommand := ""
```

Comment:
can you remove the container command entirely from the metrics?

Reply:
@rootfs, done!